tests/test_registry.py

```diff
@@ -442,6 +441,9 @@ def check_type_attr(name, expected_class, attr_name, expected_attr):
 
   check_obj_attr(r.commands['vkCreateInstance'], Command, 'successcodes',
                  'VK_SUCCESS')
+  check_obj_attr(r.commands['vkEnumerateDeviceLayerProperties'], Command,
+                 'errorcodes',
+                 'VK_ERROR_OUT_OF_HOST_MEMORY,VK_ERROR_OUT_OF_DEVICE_MEMORY')
```

Comment:
Why is the expected value a comma separated string, rather than a list of strings? Based on the code in `class Command`, it looks like this is split into a list.

Reply:
Because check_obj_attr reads the actual value from the xml_node. We can change this but this is also how the other tests work at the moment.

Reply:
Is there anywhere that is testing the parsed version of `errorcodes` added by this CL?

Reply:
Done.